astropy/coordinates/tests/test_exceptions.py

```diff
@@ -51,12 +47,22 @@ class FrameDescription(NamedTuple):
     ),
     "default_GCRS",
 )
-icrs = FrameDescription(ICRS(0 * u.deg, 0 * u.deg), "ICRS Frame", "ICRS")
+skycoord_galactic = FrameDescription(
+    SkyCoord(0 * u.deg, 0 * u.deg, frame="galactic"),
+    "Galactic Frame",
+    "SkyCoord_Galactic",
+)
+skycoord_icrs = FrameDescription(
+    SkyCoord(0 * u.deg, 0 * u.deg), "ICRS Frame", "SkyCoord_ICRS"
+)
 
 
 @pytest.mark.parametrize(
     "coord_from,coord_to",
-    [pytest.param(icrs, gcrs_custom), pytest.param(gcrs_default, galactic)],
+    [
+        pytest.param(skycoord_icrs, gcrs_custom),
+        pytest.param(gcrs_default, skycoord_galactic),
```

Comment:
Is there any upside to using `pytest.param` (without keyword arguments) instead of plain tuples that I'm missing ?
```suggestion
        (skycoord_icrs, gcrs_custom),
        (gcrs_default, skycoord_galactic),
```